src/components/BackToTop/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react'
+import { FaChevronUp } from 'react-icons/fa'
+import styles from './styles.module.scss'
+
+export default function BackToTop() {
+    const [visible, setVisible] = useState<boolean>(false);
+
+    const toggleVisible = () => {
+        const scrolled = document.documentElement.scrollTop;
+        if (scrolled > 300){
```

Comment:
declare a constant on the top of the component.

Reply:
it's related to cleaning code (avoid magic numbers). The best way to fix magic numbers is to move them into their own constant variables. This way the variable name of the magic number can describe exactly what the variable does. In this case, you can declare a constant : 
const SCROLL_THESHOLD = 300